source/_integrations/fan.markdown

```diff
@@ -9,3 +9,665 @@ ha_release: 0.27
 ---
 
 The `fan` integration is built for the controlling of fan devices.
+
+
+
+## Services
+
+### Fan control services
+
+Available services: `fan.set_speed`, `fan.turn_on`, `fan.turn_off`, `fan.oscillate`, `fan.toggle`, `fan.set_direction`, `fan.xiaomi_miio_set_buzzer_on`, `fan.xiaomi_miio_set_buzzer_off`, `fan.xiaomi_miio_set_led_on`, `fan.xiaomi_miio_set_led_off`, `fan.xiaomi_miio_set_child_lock_on`, `fan.xiaomi_miio_set_child_lock_off`, `fan.xiaomi_miio_set_favorite_level`, `fan.xiaomi_miio_set_led_brightness`, `fan.xiaomi_miio_set_auto_detect_on`, `fan.xiaomi_miio_set_auto_detect_off`, `fan.xiaomi_miio_set_learn_mode_on`, `fan.xiaomi_miio_set_learn_mode_off`, `fan.xiaomi_miio_set_volume`, `fan.xiaomi_miio_reset_filter`, `fan.xiaomi_miio_set_extra_features`, `fan.xiaomi_miio_set_target_humidity`, `fan.xiaomi_miio_set_dry_on`, `fan.xiaomi_miio_set_dry_off`, `fan.wemo_set_humidity`, `fan.wemo_reset_filter_life`, 
```

Comment:
We should not document integration/brand-specific services on the generic platform page.
So please remove those.

Reply:
Hi @frenck 

Shall I remove all that block of available services or only the ones that have a brand name?

Reply:
Yes, only services that belong to the fan integration itself should be added.

Reply:
I am a bit confused because I got all the services from this file https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/fan/services.yaml and as you can see all of those services belong to the fan integration.

Reply:
Yes, they shouldn't be there either 😉